### PR TITLE
Add file

### PR DESCRIPTION
**Pull request by jeffrey-dot-li**
Closes issue #26

### CODE REVIEW

**Comment by seendsouza** [comment]
y r there like 32 commits

File: files/testEntry.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+{"team":"8268","score":"18-2","result":"W"}
```
Comment (seendsouza) [inline files/testEntry.txt:1]: Why should this be in the repository

Comment (seendsouza) [reply files/testEntry.txt:1]: ig u could use these for tests

Comment (seendsouza) [reply files/testEntry.txt:1]: but there aren't any tests rn

File: lib/src/commands/add/file.dart [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import 'dart:io';
+import 'dart:convert';
+import 'package:args/command_runner.dart';
+import '../../utils/utils.dart';
+import 'package:path/path.dart';
```
Comment (seendsouza) [inline lib/src/commands/add/file.dart:5]: https://dart.dev/guides/language/effective-dart/style#ordering

File: lib/src/commands/add/file.dart [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+  FileCommand(databaseHandler) {
+    argParser..addFlag('verbose', defaultsTo: false);
+    this.databaseHandler = databaseHandler;
+  }
+
+  @override
+  run() async {
+    String fileFolder = '/../files/';
+//  Tries to find file in [files] folder.
```
Comment (seendsouza) [inline lib/src/commands/add/file.dart]: put this comment and the two below inline with the rest of the function

File: lib/src/commands/add.dart [hunk 4]
```diff
@@ -2,6 +2,7 @@ import 'package:args/command_runner.dart';
 
 import 'add/data.dart';
 import 'add/type.dart';
+import 'add/file.dart';
```
Comment (seendsouza) [inline lib/src/commands/add.dart:4]: https://dart.dev/guides/language/effective-dart/style#do-sort-sections-alphabetically

File: files/testEntry.txt (hunk 1 above)
Comment (seendsouza) [reply files/testEntry.txt:1]: Also not everything should be stored as a string, so you won't need quotes for everything. A list would prob be better for score

Comment (seendsouza) [reply files/testEntry.txt:1]: Or do u just wanna have things as strings and discover its type by guessing?

Comment (jeffrey-dot-li) [reply files/testEntry.txt:1]: Oh I just forgot to gitignore this file

Comment (jeffrey-dot-li) [reply files/testEntry.txt:1]: Also I will make it be able to take lists in a future pull request? or should that be here

Comment (seendsouza) [reply files/testEntry.txt:1]: yeah make than an issue @DrCheddar

**Review by seendsouza — APPROVED** [review]
Make Travis like you then u gucci

**Pull request merged**